cmd/monaco/integrationtest/v2/test-resources/integration-documents/project/document-notebook/config.yaml

```diff
@@ -7,4 +7,4 @@ configs:
   type:
     document:
       kind: notebook
-      private: true
+      private: false
```

Comment:
Would we prefer to go with the default? I.e. omit `private`

Reply:
also thought about it. Changing the value from "not present" to "private: true" in the test is then a bit more messy.
I think it is fine.